src/views/v3/Bridge/index.tsx

```diff
@@ -264,7 +266,10 @@ function Bridge(props: BridgeProps) {
 
   // Supported chains for the source network
   const supportedSourceChains = useMemo(() => {
-    return getFilteredChains(supportedChains, destChain);
+    // Prevent HyperCore from appearing as a source since it's destination-only
+    return getFilteredChains(supportedChains, destChain).filter(
```

Comment:
Might be worth to just change getFilteredChains() to take an arg saying isSource, and filter there. Otherwise we will repeat.